js/main.js

```diff
@@ -33,24 +33,63 @@
   // 削除ボタンを定義
   function addDeleteButton() {
     const deleteButton = createButton('削除');
-    deleteButton.className = 'Delete';
-    deleteButton.onclick = clickDeleteButton; // 削除ボタンClick時の処理を追加
+    deleteButton.className = 'DeleteBottunClass';

```

Comment:
軽微な指摘になりますが、以下のように修正しましょう。
```suggestion
    deleteButton.className = 'DeleteButtonClass';

```